flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/NtTable.java

```diff
@@ -0,0 +1,91 @@
+package com.flash3388.frc.nt;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+public class NtTable {
+
+    public class Builder {
+
+        private final NetworkTable mTable;
+
+        public Builder(NetworkTable table) {
+            mTable = table;
+        }
+
+        public Builder(String name) {
+            this(NetworkTableInstance.getDefault(), name);
+        }
+
+        public Builder addBooleanEntry(String name, boolean initialValue) {
+            return addEntry(name, NetworkTableValue.makeBoolean(initialValue));
+        }
+
+        public Builder(NetworkTableInstance tableInstance, String name) {
```

Comment:
I'm questioning whether or not it was on purpose

Reply:
may be

Reply:
Dude, above the `Builder(String)` one:

```Java
        public Builder(NetworkTable table) {
            mTable = table;
        }

       public Builder(NetworkTableInstance tableInstance, String name) {
           this(instance.getTable(name));
       }

        public Builder(String name) {
            this(NetworkTableInstance.getDefault(), name);
        }
```

Reply:
oh really?